doc/04-Upgrading.md

```diff
@@ -3,6 +3,65 @@
 Specific version upgrades are described below. Please note that version upgrades are incremental.
 If you are upgrading across multiple versions, make sure to follow the steps for each of them.
 
+## Upgrading to Icinga DB v1.1.2
+
+Please apply the `1.1.2.sql` upgrade script to your database. For package installations, you can find this file at
+`/usr/share/icingadb/schema/mysql/upgrades/` or `/usr/share/icingadb/schema/pgsql/upgrades/`, depending on your
+database vendor.
+
+!!! warning
+
+    Please ignore this warning if you are using PostgreSQL. This applies to MySQL/MariaDB users only.
+
+    In order to fix a loading performance issue of the history view in Icinga DB Web, this upgrade script adds an
+    appropriate index on the `history` table. Creating this new index normally takes place without blocking any other
+    queries. However, this may hang for a relatively considerable time, blocking all Icinga DB queries on all
+    `*_history` tables and the `history` table inclusively if there is an ongoing, long-running query on the `history`
+    table. One way of causing this to happen is if an Icinga Web user accesses the `icingadb/history` view just before
+    you are running this script. Depending on how many entries you have in the history table, Icinga DB Web may take
+    quite a long time to load, until your web servers timeout (if any) kicks in. 
+    See [unblock history tables](#unblock-history-tables) for details.
```

Comment:
I think this is quite lengthy for a warning block given that there's a separate section for the details already.

My suggestion for the warning block:

> With MySQL and MariaDB, a locking issue can occur if the schema upgrade is applied while the history view is accessed in Icinga DB Web. This can result in the upgrade being delayed unnecessarily and it blocking other queries. Please see unblock history tables for how to detect and resolve this situation.

All the details that are removed by this change can be moved to the section below.

Reply:
> Please ignore this warning if you are using PostgreSQL. This applies to MySQL/MariaDB users only.

Does your suggestion include this hint? as well?

Reply:
For me, "With MySQL and MariaDB," would be enough of a hint that this does not affect PostgreSQL. I think you added that line in response to @oxzi, so let's ask him if this suffices or whether it should be more explicit about PostgreSQL. If I remember correctly, it previously didn't mention a database type at all (apart from in the prompts and implicitly by using MySQL/MariaDB-specific commands).